geonode_mapstore_client/client/js/plugins/MapViewersCatalog.jsx

```diff
@@ -32,17 +33,23 @@ function MapViewersCatalogPlugin({
     match,
     resourcesParams,
     location,
+    defaultViewerPlugins = ['Annotations', 'TOC', 'BackgroundSelector', 'Identify', 'QueryPanel', 'Measure', 'Print', 'MousePosition', 'Search', 'ScaleBox', 'GlobeViewSwitcher', 'ZoomAll', 'ZoomIn', 'ZoomOut', 'Timeline', 'MetadataExplorer', 'Widgets'],
```

Comment:
@dsuren1 shouldn't this come from the list of plugins we configure inside `localConfig` for the `map_viewer`? We will need to keep them aligned manually otherwise.

Reply:
Yes, these are defaults and you can still configure these from localConfig `MapViewersCatalog` plugin cfg